src/AtlasModules/atlas-config.bat

```diff
@@ -2216,7 +2223,9 @@ Reg.exe delete "HKCR\DesktopBackground\Shell\NVIDIAContainer" /f
 :: delete icon exe
 erase /F /Q "C:\Windows\System32\NvidiaIcon.exe"
 taskkill /f /im explorer.exe
-nsudo -U:C start explorer.exe
+taskkill /f /im explorer.exe >nul 2>&1
+taskkill /f /im explorer.exe >nul 2>&1
+nsudo.exe -U:E -P:E -Wait explorer.exe
```

Comment:
This seems to error when launching?

Reply:
Are you testing it with trustedinstaller permissions? 
```
nsudo.exe -U:C explorer.exe
```
Works fine

Reply:
I was testing it with `-U:E -P:E` (current user elevated, all perms), but the command you posted above also works fine for me, so I will change it. Thanks